go.mod

```diff
@@ -3,23 +3,23 @@ module github.com/lightninglabs/pool
 go 1.14
 
 require (
-	github.com/btcsuite/btcd v0.21.0-beta.0.20201208033208-6bd4c64a54fa
+	github.com/btcsuite/btcd v0.21.0-beta.0.20210429225535-ce697fe7e82b
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/btcsuite/btcutil v1.0.2
-	github.com/btcsuite/btcwallet v0.11.1-0.20201207233335-415f37ff11a1
+	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
+	github.com/btcsuite/btcwallet v0.12.0
 	github.com/btcsuite/btcwallet/wallet/txrules v1.0.0
-	github.com/btcsuite/btcwallet/wtxmgr v1.2.0
+	github.com/btcsuite/btcwallet/wtxmgr v1.3.0
 	github.com/davecgh/go-spew v1.1.1
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.4.3
 	github.com/grpc-ecosystem/grpc-gateway v1.14.6
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/aperture v0.1.6-beta
-	github.com/lightninglabs/lndclient v0.12.0-3
-	github.com/lightninglabs/pool/auctioneerrpc v1.0.0
-	github.com/lightninglabs/protobuf-hex-display v1.3.3-0.20191212020323-b444784ce75d
-	github.com/lightningnetwork/lnd v0.12.0-beta
+	github.com/lightninglabs/lndclient v0.12.0-7
+	github.com/lightninglabs/pool/auctioneerrpc v1.0.2
+	github.com/lightninglabs/protobuf-hex-display v1.4.3-hex-display
+	github.com/lightningnetwork/lnd v0.13.0-beta.rc2
```

Comment:
We should be cutting this next release before 0.13 is finalzied, but I think that's ok, we'll just need to circle back and update things if any critical API level stuff has changed.